WcaOnRails/app/jobs/update_wca_registration.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+#
+require_relative '../helpers/microservices_helper'
+class UpdateWCARegistration < ApplicationJob
+  def perform(status)
+    token = get_wca_token
+    response = httparty.post(update_payment_status_path, headers: { 'X-WCA-Service-Token' => token, "Content-Type" => "application/json" }, body: { payment_status: status }.to_json)
+    unless response.ok?
+      throw Error "Updating wca-registration failed with error #{response.status} #{response.body}"
```

Comment:
Shouldn't it be `raise` in Ruby?!

Reply:
[throw](https://apidock.com/ruby/v2_5_5/Kernel/throw) actually works too, but I think just for legacy reasons which why my ide didn't complain